openapi.yaml

```diff
@@ -3713,6 +3728,10 @@ components:
               minItems: 1
               items:
                 type: string
+    incomplete_result:
+      type: boolean
+      default: false
+      description: MUST be `true` when the batch job result has not finished yet and as such is potentially incomplete.
```

Comment:
See above: "finished" -> "finished successfully"

Reply:
Will check the wording and/or make clear that this corresponds to the job status enum.

Reply:
This terminology is not present anymore after switching to `openeo:status`.